app/src/main/java/arjun/hanabiandroid/game/GameState.java

```diff
@@ -0,0 +1,25 @@
+package arjun.hanabiandroid.game;
+
+import java.util.ArrayList;
+import java.util.Queue;
+
+public class GameState {
+    final int turnOrder;
+    final Queue<Card> deck;
+    final ArrayList<Card> discard = new ArrayList<>();
+    final Hand[] hands;
+    final Card[] piles;
+    int cluesRemaining;
+    int mistakesRemaining;
+
+    public GameState(InitialState initialState) {
+        this.turnOrder = initialState.turnOrder;
+        this.deck = DeckGenerator.generateDeck(initialState);
```

Comment:
I'm not sure what you mean. I can see `DeckGenerator` in the list of files in this PR.